### PR TITLE
Add manifests

### PR DESCRIPTION
**Pull request by funkypenguin**

### CODE REVIEW

File: deploy/manifests/agent/portainer-agent-edge-k8s.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+metadata:
+  name: portainer-agent
+  namespace: portainer
+spec:
+  clusterIP: None
+  selector:
+    app: portainer-agent
+---
+apiVersion: apps/v1
+kind: DaemonSet
```
Comment (deviantony) [inline deploy/manifests/agent/portainer-agent-edge-k8s.yaml:52]:
```suggestion
kind: Deployment
```

File: deploy/manifests/agent/portainer-agent.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+metadata:
+  name: portainer-agent-headless
+  namespace: portainer
+spec:
+  clusterIP: None
+  selector:
+    app: portainer-agent
+---
+apiVersion: apps/v1
+kind: DaemonSet
```
Comment (deviantony) [inline deploy/manifests/agent/portainer-agent.yaml:51]:
```suggestion
kind: Deployment
```

File: deploy/manifests/agent/portainer-edge-agent-setup.sh [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    error "Usage: ${0} <EDGE_ID> <EDGE_KEY>"
+    exit 1
+  fi
+
+  [[ "$(command -v curl)" ]] || errorAndExit "Unable to find curl binary. Please ensure curl is installed before running this script."
+  [[ "$(command -v kubectl)" ]] || errorAndExit "Unable to find kubectl binary. Please ensure kubectl is installed before running this script."
+
+  info "Downloading agent manifest..."
+  curl -L https://portainer.github.io/k8s/deploy/manifests/agent/portainer-agent-edge-k8s.yaml -o portainer-agent-edge-k8s.yaml || errorAndExit "Unable to download agent manifest"
+
```
Comment (deviantony) [inline deploy/manifests/agent/portainer-edge-agent-setup.sh:59]: Missing instructions to add to this script:

```
info "Creating Portainer namespace..."
kubectl create namespace portainer
```

**Review by deviantony — CHANGES_REQUESTED** [review]
Few changes required regarding the agent

File: deploy/manifests/agent/portainer-edge-agent-setup.sh (hunk 3 above)
Comment (funkypenguin) [reply deploy/manifests/agent/portainer-edge-agent-setup.sh:59]: @deviantony said he'll resolve this one :)

File: deploy/manifests/agent/README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+To deploy an Edge agent inside your Kubernetes cluster, it is recommended to follow the instructions available inside your Portainer instance.
+
+# Usage
+
+## Deploy the Portainer agent and access it via an external load balancer
+
+If your cloud provider supports external load balancers, you can use the following command to deploy the regular Portainer agent (not Edge):
+
+```
+kubectl ... apply -f portainer-agent.yaml
```
Comment (deviantony) [inline deploy/manifests/agent/README.md:14]: Should probably use the new filename here.

**Review by deviantony — CHANGES_REQUESTED** [review]
Overall LGTM, minor change required.

File: deploy/manifests/agent/README.md (hunk 4 above)
Comment (funkypenguin) [reply deploy/manifests/agent/README.md]: Fixed in latest commit :)

**Review by deviantony — APPROVED** [review]
LGTM

**Pull request merged**